packages/bot-web-ui/src/app/app.tsx

```diff
@@ -0,0 +1,25 @@
+import '../public-path'; // Leave this here (at the top)! OK boss!
+import React from 'react'; // eslint-disable-line import/first
```

Comment:
```suggestion
import React from 'react';
```
@shafin-deriv Not related to your changes, But seems like we can remove it 🤔